src/tools/firebase.ts

```diff
@@ -0,0 +1,22 @@
+import { getAnalytics, logEvent } from "firebase/analytics";
+
+// Import the functions you need from the SDKs you need
+import { initializeApp } from "firebase/app";
+// TODO: Add SDKs for Firebase products that you want to use
+// https://firebase.google.com/docs/web/setup#available-libraries
+
+// Your web app's Firebase configuration
+// For Firebase JS SDK v7.20.0 and later, measurementId is optional
+const firebaseConfig = {
+  apiKey: "AIzaSyBN90GcGDAqgC6OFpGzv8Zt9lc41txCQI8",
```

Comment:
is it safe to have the value for the apiKey here? just curious, mostly I'm using env file for these